ui/src/main/scripts/plugins/directives/instance-view-diagram/ts/instanceViewDiagramDirective.ts

```diff
@@ -194,9 +194,13 @@ module InstanceViewDiagram {
           label += '<div class="name">' + uri + '</div>';
 
           let nodeTT = '<ul>';
+          let hasFault = false;
           _.each(d.properties, (property) => {
             if (!!property.value) {
               nodeTT += ('<li class=\'tt-prop\'><strong>' + property.name + '</strong> ' + property.value + '</li>');
+              if (property.name === 'fault') {
```

Comment:
I know we are using `fault` and `fault.[code|description]` for errors, but OT defines `error` tag
http://opentracing.io/documentation/pages/api/data-conventions.html#errors

we should discuss this when @objectiser is back. (It can be handled in separate jira)